src/util/prolog/prologResult.ts

```diff
@@ -51,12 +65,22 @@ export class PrologResult {
 		return results;
 	}
 
+	/**
+		* Receives the filled-in values for a single variable
+		* @param variable - The variable to fill in
+		* @return Fill in values in order of their return
+		*/
 	private getResultsFor(variable: string): string[] {
 		// Group 1 will be the content of the variable independent of the variable length.
 		const pattern = new RegExp(variable + variableAnswerPattern, "g");
 		return fetchAllMatchesForAGroup(this.rawResults.toString(), pattern, 1);
 	}
 
+	/**
+		* Curates and raw results so every variable has their own field and only their fill in values are given
```

Comment:
Is there a word missing here? And what are fill in values?

Reply:
Fill-in values are supposed to be what prolog returns as a possible variable value to make a predicate true. It could seem like an assignment, but strictly speaking it isn't so I tried maybe too hard to avoid that term.